### PR TITLE
plat-rcar: disable HW RNG by default (and try to workaround the issue)

### PR DESCRIPTION
**Pull request by lorc**
During repetitive testing I noticed that sometimes OP-TEE panics due to problems with Renesas HW random number generator. While proposed workaround seems to help, I believe it is better to disable this feature by default, at least until I can get some clarification from Renesas side, which can take quite time.

### CODE REVIEW

File: core/arch/arm/plat-rcar/romapi.c [hunk 1]
```diff
@@ -129,7 +130,13 @@ uint32_t plat_rom_getrndvector(uint8_t rndbuff[PLAT_RND_VECTOR_SZ],
 	assert(rndbuff_pa % RCAR_CACHE_LINE_SZ == 0);
 	assert(scratch_pa % RCAR_CACHE_LINE_SZ == 0);
 
-	ret = plat_call_romapi(func_addr, rndbuff_pa, scratch_pa, scratch_sz);
+	while (try++ < 3) {
+		ret = plat_call_romapi(func_addr, rndbuff_pa, scratch_pa,
+				       scratch_sz);
+		if (ret == 0)
+			break;
+		IMSG("ROM_GetRndVector() returned 0x%X", ret);
```
Comment (jenswi-linaro) [inline core/arch/arm/plat-rcar/romapi.c:138]: `"%#"PRIx32`, gives `0x` prefix automatically and `PRIx32` for a 32-bit type using lower case letters.

**Review by jenswi-linaro — COMMENTED** [review]
With my comment addressed, please apply:
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Review by etienne-lms — COMMENTED** [review]
`Acked-by: Etienne Carriere <etienne.carriere@foss.st.com>`

**Comment by lorc** [comment]
Thank you. Tags are applied.

**Pull request merged**